src/in-depth/config-files.md

```diff
@@ -0,0 +1,47 @@
+# Using config files
+
+Dealing with configurations can be annoying
+especially if you support multiple operating systems
+which all have their own places
+for short- and long-term files.
+
+There are multiple solutions to this,
+some being more low-level than others.
+
+The easiest crate to use for this is `confy`.
+It asks you for the name of your application
+and requires you to specify the config layout
+via a `struct` (that is `Serialize`, `Deserialize`)
+and it will figure out the rest!
+
+```rust,ignore
+#[derive(Debug, Serialize, Deserialize)]
+struct MyConfig {
+    name: String,
+    comfy: bool,
+    foo: i64,
+}
+
+fn main() -> Result<(), io::Error> {
+    let cfg: ConfyConfig = confy::load("my_app")?;
+    println!("{:#?}", cfg);
```

Comment:
I think we could have used `dbg!()` here instead after the debug have become stable?